lib/helpers/getMessage.js

```diff
@@ -35,12 +37,22 @@ module.exports = function getMessage(message) {
                 var part = {
                     which: info.which,
                     size: info.size,
-                    body: body
                 };
 
                 if (isHeader.test(part.which)) {
-                    part.body = Imap.parseHeader(part.body);
-                }
+                    part.body = Imap.parseHeader(body);
+		} else {
+		    if (!encoding)
+		        part.body = body;
+	            else if (encoding === 'BASE64')
+                        part.body = new Buffer(body, 'base64');
+                    else if (encoding === 'QUOTED-PRINTABLE')
+                        part.body = new Buffer(qp.decode(body));
+                    else if (encoding === '7BIT')
+                        part.body = new Buffer(body); // that's fine
+                    else
+                        throw Error('Unknown encoding ' + encoding);
```

Comment:
Would rather just leave it serialized instead of throwing here. User can figure out how to decode it for themselves if its something the library isn't automatically handling.